crates/bevy_pbr/Cargo.toml

```diff
@@ -5,20 +5,26 @@ edition = "2021"
 description = "Adds PBR rendering to Bevy Engine"
 homepage = "https://bevyengine.org"
 repository = "https://github.com/bevyengine/bevy"
-license = "MIT OR Apache-2.0"
+license = "MIT"
```

Comment:
In the ex-`pipelined` crates the LICENSE is `MIT` instead of `MIT OR Apache-2.0`

Reply:
Ooh good catch